sips/sip-031/sip-031.md

```diff
@@ -0,0 +1,575 @@
+
+### **Stacks Roadmap Summary**
+
+This treasury will fuel development along the [Stacks roadmap](http://stacksroadmap.com), which has been refreshed via a series of [community workshops](https://forum.stacks.org/t/stacks-roadmap-draft-final-call-community-feedback/17984) ([1](https://forum.stacks.org/t/stacks-roadmap-draft-final-call-community-feedback/17984)), the latest version of which has been published in support of this SIP. The treasury will be created to deliver on these and other priorities as defined by builders and key stakeholders and provide the necessary resources for Stacks’ progression in the following critical areas, among others:
+
+**1\. Stacks Core Development:**
+
+Upgrades to increase transaction speed, reliability, scalability, and security — including improved block production, miner replay logic, Clarity-to-Wasm compilation, ongoing enhancements to PoX and stacking UX, security bounties, audits, and monitoring.
+
+**2\. sBTC Protocol & Stacks Value Accrual:**
+
+Treasury funding will also support the development of new mechanisms that drive STX demand such as dual stacking, bitcoin payments use cases, and BTC yield strategies — as well as continued development of critical sBTC features like self-custodial minting and upgrades for further trust-minimization. By funding these areas, the Endowment aims to create a self-reinforcing flywheel where protocol revenue grows treasury reserves → reserves fund development and adoption → adoption increases protocol usage and STX demand → drives continued revenue growth. This model provides a sustainable path for long-term operations and Stacks ecosystem growth.
+
+**3\. Ecosystem Growth:**
+
+Strategic efforts to scale TVL, liquidity, and user adoption through community based marketing and growth strategies, grant programs, DeFi incentives, sBTC TVL milestones, stablecoin integrations, and centralized exchange listings for SIP-10 tokens.
+
+This roadmap reflects the core areas where capital from the Stacks endowment will be deployed if SIP-031 is approved — focusing on core protocol upgrades, institutional support, developer traction, and Stacks value accrual.
+
+### Key artifacts related to this SIP
+
+In preparing this SIP, the authors have gathered essential information to help Stacks holders provide informed feedback and make confident voting decisions. Inside, you’ll find summaries and links to:
+
+* A complete economic impact study, performed by a neutral 3rd party token economics research firm. (LINK)  
+* The latest iteration of the Stacks Roadmap, containing the sum total of feedback and input from the community over the past 3-months for core development on Stacks supporting a range of use cases ([stacksroadmap.com](https://stacksroadmap.com/))  
```

Comment:
Grammar: "3-months" is not hyphenated

Reply:
following my suggestion above, if links are annotated it could be referred to here as `[1]` for example.

Reply:
```suggestion
* The latest iteration of the Stacks Roadmap, containing the sum total of feedback and input from the community over the past 3 months for core development on Stacks supporting a range of use cases ([stacksroadmap.com](https://stacksroadmap.com/))  
```